proxy/envoy/route.go

```diff
@@ -31,10 +31,10 @@ import (
 
 const (
 	// InboundClusterPrefix is the prefix for service clusters co-hosted on the proxy instance
-	InboundClusterPrefix = "in:"
+	InboundClusterPrefix = "in."
```

Comment:
I'll revisit it next week, once Matt decides what the valid cluster names are. We'll need to rename all clusters with synthetic short names anyways.